DeepSpeech.py

```diff
@@ -1444,15 +1445,18 @@ def end(self, session):
     if FLAGS.summary_secs > 0:
         hooks.append(tf.train.SummarySaverHook(save_secs=FLAGS.summary_secs, output_dir=FLAGS.summary_dir, summary_op=merge_all_summaries_op))
 
+    # Hook wih number of checkpoint files to save in checkpoint_dir
+    if FLAGS.max_to_keep > 0:
+	saver_var = tf.train.Saver(max_to_keep=FLAGS.max_to_keep)
```

Comment:
`saver_var` is a strange name - please change it so something else.
Also: indenting